src/devices/cavius.c

```diff
@@ -0,0 +1,163 @@
+/** @file
+*   Cavius smoke alarm decoder.
+*
```

Comment:
The `@file` comment should contain the copyright and licence. The next `/**` comment should document the decoder function.

Reply:
Fixed

Reply:
`/** @file` not `/*** @file`
and there can be only one comment block (`/**`) for the decoder function, indent is 0 spaces.

Reply:
As far as I can tell I have complied with this now